edgar-app/src/components/connectionPage/login/modal/ResetPasswordHandler.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button, Input, Text, VStack, useToast } from '@chakra-ui/react';
+import { useForm } from 'react-hook-form';
+
+import ModalHandler from 'components/modals/ModalHandler';
+import ErrorMessage from 'components/forms/ErrorMessage';
+
+import ShieldIllustration from 'assets/illustrations/ShieldIllustration';
+
+import type { ResetPasswordType } from 'types/login/ResetPasswordType';
+
+import { useMissingPasswordMutation } from 'services/request/account';
+
+const ResetPasswordHandler = ({ isOpen, onClose }: { isOpen: boolean; onClose: () => void }): JSX.Element => {
+	const [triggerMissingPassword] = useMissingPasswordMutation();
+	const {
+		handleSubmit,
+		formState: { errors },
+		register,
+		reset,
+	} = useForm<ResetPasswordType>({
+		mode: 'onChange',
+	});
+
+	const toast = useToast({ duration: 3000, isClosable: true });
+
+	const onCloseAction = () => {
+		reset();
+		onClose();
+	};
+
+	const onSubmit = handleSubmit((data) => {
+		if (data.email) {
```

Comment:
I don't think this is useful because the handleSubmit function check if the values followed the rules